main/webservices/api/tests/UsernameExistTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [End of line character is invalid; expected "'\n'" but found "\n"](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4717817)